tests/rptest/tests/random_node_operations_test.py

```diff
@@ -217,23 +242,31 @@ def verify(self):
             del self.consumer
 
             # Start a new consumer to read all data written
-            self._start_consumer()
+            self._start_consumer(with_logs=True)
             self.consumer.wait()
-            if self.topic.cleanup_policy != TopicSpec.CLEANUP_COMPACT:
+            if not self.compaction_enabled:
                 assert self.consumer.consumer_status.validator.valid_reads >= self.producer.produce_status.acked, f"Missing messages from topic: {self.topic}. valid reads: {self.consumer.consumer_status.validator.valid_reads}, acked messages: {self.producer.produce_status.acked}"
 
             assert self.consumer.consumer_status.validator.invalid_reads == 0, f"Invalid reads in topic: {self.topic}, invalid reads count: {self.consumer.consumer_status.validator.invalid_reads}"
 
     @skip_debug_mode
     @cluster(num_nodes=7,
              log_allow_list=CHAOS_LOG_ALLOW_LIST + PREV_VERSION_LOG_ALLOW_LIST)
-    @matrix(enable_failures=[False, True],
+    @matrix(enable_failures=[True, False],
             num_to_upgrade=[0],
```

Comment:
Not from this PR, but this was [0,3] before https://github.com/redpanda-data/redpanda/pull/9869. Was that intentional or just to facilitate local dockerized runs? Shouldwe add this back?

Reply:
yeah, we need to add this back